#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'hsr_task_common'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()
gen.add("Hour",              int_t,    0,
        "",
         0, 0, 24)
gen.add("Minute",              int_t,    0,
        "",
         0, 0, 60)
gen.add("call_msg",             str_t,    0,
        "call_msg",
        "")
gen.add("message",             str_t,    0,
        "message",
        "Welcome Home!")

exit (gen.generate (PACKAGE, "hsr_task_common", "AdditionalRqt"))
